lib/tasks/development.rb

```diff
@@ -36,8 +36,10 @@ def init_ctr_user
     return if File.exist?(file)
 
     require 'io/console'
-    puts 'Enter password:'
-    plain_password = $stdin.noecho(&:gets).chomp
+    plain_password = ENV.fetch('OOD_DEV_CONTAINER_PASS') {
+      puts 'Enter password:'
+      $stdin.noecho(&:gets).chomp
+    }
```

Comment:
Can we try checking for `$stdin.isatty` here instead? Maybe not a security thing, but it just seems more straightforward then wondering why we'd use/have an ENV variable.

I mean in a year or more, it'll be apparent that if it isn't a tty, obviously it can't prompt so we just force a password to be set.